nbconvert/exporters/html.py

```diff
@@ -98,22 +99,22 @@ def _template_name_default(self):
     def default_config(self):
         c = Config({
             'NbConvertBase': {
-                'display_data_priority' : ['application/vnd.jupyter.widget-state+json',
```

Comment:
I removed this as I don't see why this would be in cell outputs? I can revert this change if it can actually happens

Reply:
+1 for removing this. It does not make sense.